Documentation/network/servicemesh/istio.rst

```diff
@@ -25,8 +25,9 @@ Istio's `sidecar proxies <https://istio.io/latest/docs/ops/deployment/architectu
 Disruptions can happen when you enable Cilium's ``kubeProxyReplacement`` feature  (see :ref:`kubeproxy-free` docs), 
 which enables socket based load balancing inside a Pod.
 
+
 To ensure that Cilium does not interfere with Istio, Cilium must be deployed
-with the ``--config bpf-lb-sock-hostns-only=true`` cilium CLI flag or with the ``socketLB.hostNamespaceOnly`` Helm value.
+with the ``--set socketLB.hostNamespaceOnly=true`` cilium CLI flag, or with the ``socketLB.hostNamespaceOnly`` Helm value.
```

Comment:
Given that the `cilium` CLI now defaults to Helm mode, we can maybe avoid giving twice the same value name? How about:

```suggestion
with the ``socketLB.hostNamespaceOnly`` Helm value set to ``true``.
```

Another (possibly better) option is to keep `bpf-lb-sock-hostns-only` as we have now, but to tell that it's the configmap value rather than the CLI flag.

Reply:
OK. Rewrote and kept the params.